src/registry_schemas/schemas/party.json

```diff
@@ -119,7 +120,48 @@
                 "officer",
                 "roles",
                 "mailingAddress"
-            ]
+            ],
+            "if": {
```

Comment:
In the schema, we can just make the lastName mandatory, and everything else optional.
As eventually that will come in to affect for all natural persons.